cli/cmd/org/rename.go

```diff
@@ -26,65 +27,92 @@ func RenameCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			fmt.Println("Warn: Renaming an org would invalidate dashboard URLs")
+			if !cmd.Flags().Changed("new-name") && !cmd.Flags().Changed("display-name") {
+				return fmt.Errorf("at least one of --new-name or --display-name must be provided")
+			}
 
-			if !cmd.Flags().Changed("org") && ch.Interactive {
+			if !cmd.Flags().Changed("org") && len(args) == 0 && ch.Interactive {
 				orgNames, err := OrgNames(ctx, ch)
 				if err != nil {
 					return err
 				}
 
-				name, err = cmdutil.SelectPrompt("Select org to rename", orgNames, "")
+				name, err = cmdutil.SelectPrompt("Select org to edit", orgNames, ch.Org)
 				if err != nil {
 					return err
 				}
 			}
 
-			if ch.Interactive {
-				err = cmdutil.SetFlagsByInputPrompts(*cmd, "new-name")
-				if err != nil {
-					return err
+			resp, err := client.GetOrganization(ctx, &adminv1.GetOrganizationRequest{Name: name})
+			if err != nil {
+				if st, ok := status.FromError(err); ok && st.Code() == codes.NotFound {
+					return fmt.Errorf("org %q doesn't exist, run 'rill org list' to see available orgs", name)
 				}
+				return err
 			}
+			org := resp.Organization
 
-			if newName == "" {
-				return fmt.Errorf("please provide valid org new-name, provided: %q", newName)
+			// Require at least one change
+			if !cmd.Flags().Changed("new-name") && !cmd.Flags().Changed("display-name") {
+				return fmt.Errorf("at least one of --new-name or --display-name must be provided")
 			}
 
-			if !force {
-				msg := fmt.Sprintf("Do you want to rename org \"%s\" to \"%s\"?", color.YellowString(name), color.YellowString(newName)) // nolint:gocritic // Because it uses colors
-				ok, err := cmdutil.ConfirmPrompt(msg, "", false)
-				if err != nil {
-					return err
-				}
-				if !ok {
-					return nil
+			// Build update request
+			req := &adminv1.UpdateOrganizationRequest{
+				Name: org.Name,
+			}
+
+			if cmd.Flags().Changed("new-name") {
```

Comment:
When you have `if cmd.Flags().Changed("...")` two places for the same name, the code can probably be simplified. For example, you have `cmd.Flags().Changed("new-name")` both here and on line 56. Instead, you could have (pseudocode):
```go
var flagSet bool
req := &adminv1.UpdateOrganizationRequest{ ... }

if cmd.Flags().Changed("new-name") {
	flagSet = true
	// ...
}

if cmd.Flags().Changed("display-name") {
	flagSet = true
	// ...
}

if !flagSet {
	return  fmt.Errorf("must specify at least one update flag")
}
```

Reply:
This comment applies in several files throughout this PR. Please go over the diff and look for all cases where it applies.